recipes/thorvg/all/conanfile.py

```diff
@@ -141,6 +152,7 @@ def generate(self):
             tc.project_options["simd"] = bool(self.options.with_simd)
         else:
             tc.project_options["vector"] = bool(self.options.with_vector)
+        tc.project_options["extra"] = str(self.options.with_extra) if self.options.get_safe("with_extra") else ''
```

Comment:
Sorry, @toge @RubenRBS I just updated your comment and I was wrong:

```suggestion
        if self.options.get_safe("with_extra"):
            tc.project_options["extra"] = str(self.options.with_extra) or ''
```